core/export/exporter.go

```diff
@@ -235,6 +241,42 @@ func (m *Exporter) MakeExportFilesZip(svcs *services.APIServices, outfileNamePre
 			return nil, err
 		}
 	}
+	if wantDiffractionPeak {
+		fmt.Println(files.diffraction.Title)
+		fmt.Printf("%v Locations with Peaks found\n", len(files.diffraction.Locations))
+		csv, err := os.Create(path.Join(outDir, fileNamePrefix+"-diffraction-peaks.csv"))
```

Comment:
does this file need to be closed after all the writing??

Reply:
good catch this is Ryan's code I'll add it in